languages/ruby/exercises/concept/floating-point-numbers/.docs/hints.md

```diff
@@ -0,0 +1,28 @@
+### General
+
+To call methods directly on a class (as opposed to calling them on an instance of a class), you should define them with either a `self.` prefix in their name after the `def` keyword, or put the method definition between `class << self` and `end`:
```

Comment:
I was thinking about this and I'm not sure if this should be mentioned here. I would assume class methods will be covered at this point in the track. Otherwise this exercise might have too many new concepts (`floats`, `while` and now `class methods`). Someone might even say that the focus wouldn't be floating-point numbers anymore.

Reply:
Check the Basic exercise and see if class methods are mentioned there, or otherwise covered.

Reply:
The Basic exercise does not mention class methods, neither does the [String](https://github.com/exercism/v3/blob/master/languages/ruby/exercises/concept/strings/.meta/example.rb) exercise, but the String exercise does use class methods and assumes the student already knows about them. That's a bit odd since the string exercise should have no prerequisites. Not sure what to do. I don't think that the `basics` exercise is the right place to add another thing like class methods, but having an exercise only for class methods seems like an overkill to me. Maybe I should remove the explanation and not mention class methods at all. The [starting point for the exercise](https://github.com/exercism/v3/blob/8719fc420dcc6e1aec78021fb074932518814e83/languages/ruby/exercises/concept/floating-point-numbers/savings_account.rb) has them already implemented.

Reply:
We don't have to mention it if the scaffolding is there, would be my take on that.  It is a 'given'.

The basics exercise will likely be an "odd duck", it should be a primer for any information that needs to be known in order to start doing the track.  (I could be wrong here).  @iHiD or @ErikSchierboom can confirm my advice here).  We could look at the C# track to get hints of what to do here, as that seems to be the place to go for inspiration.  I recommend watching the [Anatomy of a Concept Exercise video](https://youtu.be/gkbBqd7hPrA) to see if there is clarification there as well.

Reply:
Oh, you're absolutely right. It's explained in the video. Thanks